lib/str.js

```diff
@@ -24,16 +24,19 @@ const Str = {
     },
 
     /**
-     * Converts a USD string into th number of cents it represents.
+     * Converts a currency string into the number of cents it represents.
      *
-     * @param {String}  amountStr     A string representing a USD value.
+     * @param {String}  amountStr     A string representing a currency symbol and value, like $4.02 or Bs.S97.9
      * @param {Boolean} allowFraction Flag indicating if fractions of cents should be
-     *                               allowed in the output.
+     *                                allowed in the output.
      *
      * @return {Number} The cent value of the @p amountStr.
      */
     fromUSDToNumber(amountStr, allowFraction) {
```

Comment:
Before we merge, we should consider if we want to rename this method since it now does double-duty by handling all types of currency strings (symbols + values) instead of just USD.

Reply:
Yeah I would. `fromCurrencyToNumber` maybe? Just need to make sure we replace it everywhere